README.md

```diff
@@ -0,0 +1,33 @@
+# What is Priiloader #
+
+[Priiloader](http://wiibrew.org/wiki/Preloader/Priiloader) is [a modded version of Preloader 0.30](http://wiibrew.org/wiki/Preloader) by [DacoTaco](http://wiibrew.org/wiki/User:DacoTaco) and BadUncle.
```

Comment:
the introduction could also be rewritten to explain what priiloader actually is ( an app that sits in front of system menu to allow different functionality and banner brick protection )
mention preloader that it was the base of priiloader and that this is the spiritual successor or something?